options/locale/locale_de-DE.ini

```diff
@@ -1400,6 +1400,8 @@ branch.deleted_by=Von %s gelöscht
 branch.restore_success=Branch „%s“ wurde wiederhergestellt.
 branch.restore_failed=Wiederherstellung des Branches „%s“ fehlgeschlagen.
 branch.protected_deletion_failed=Branch „%s“ ist geschützt und kann nicht gelöscht werden.
+branch.download=Branch „%s“ herunterladen
```

Comment:
Do not add translations directly in code, it needs to be done using crowdin later

Reply:
https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#translation got it